### PR TITLE
Keeping the code style consistent

### PR DESCRIPTION
**Pull request by hch12907**
- Replaced some(maybe all?) instances of `jumbledcase` with `snake_case`
- Renamed some GS structs (`GS_message`, for example)
- Renamed `camelCaseFuncs()` to `snake_case_funcs()`
- Cases where jumbledcase and snake case are mixed together (e.g. `ebit_delay_slot`) are kept as-is.

I haven't touched the IOP and GUI parts yet. (even if I did, it was done to get the code to compile)
edit: So I took a look at IOP. Not sure if I want to rename IOP_Interpreter etc to IOPInterpreter.... (or IopInterpreter?)

Hopefully the changes are acceptable.

### CODE REVIEW

File: src/core/ee/dmac.hpp [hunk 1]
```diff
@@ -9,7 +9,7 @@ struct DMA_Channel
 {
     uint32_t control;
     uint32_t address;
-    uint32_t quadword_count;
+    uint32_t quad_word_count;
```
Comment (PSI-Rockin) [inline src/core/ee/dmac.hpp]: Quadword is a single word, so there should be no extra underscore.

File: src/core/gsregisters.cpp [hunk 2]
```diff
@@ -60,11 +60,11 @@ void GS_REGISTERS::write64_privileged(uint32_t addr, uint64_t value)
             printf("MAGV: %d\n", DISPLAY2.magnify_y);
             break;
         case 0x00E0:
-            printf("[GS_r] Write BGCOLOR: $%08X\n", value);
+            printf("[GS_r] Write BGCOLOR: $%08lX\n", value);
             BGCOLOR = value & 0xFFFFFF;
             break;
         case 0x1000:
-            printf("[GS_r] Write64 to GS_CSR: $%08X_%08X\n", value >> 32, value);
+            printf("[GS_r] Write64 to GS_CSR: $%08X_%08X\n", static_cast<uint32_t>(value >> 32), static_cast<uint32_t>(value));
```
Comment (PSI-Rockin) [inline src/core/gsregisters.cpp]: Instead of using `static_cast`, use the `$%08lX` format, as it's shorter and easier to read.

**Comment by hch12907** [comment]
Made the changes you requested.

**Comment by PSI-Rockin** [comment]
Gonna merge this after sitting on this for a while. If there's any more inconsistencies in the codebase, feel free to fix them.

**Pull request merged**